dbt/adapters/bigquery/impl.py

```diff
@@ -907,3 +920,14 @@ def python_submission_helpers(self) -> Dict[str, Type[PythonJobHelper]]:
             "cluster": ClusterDataprocHelper,
             "serverless": ServerlessDataProcHelper,
         }
+
+    @classmethod
+    def render_raw_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
+        """Render the given constraint as DDL text."""
+        if constraint.type == ConstraintType.primary_key:
+            return "primary key not enforced"
```

Comment:
We should render the original constraint by calling the superclass function here as well as in the FK case.